barretenberg/cpp/src/barretenberg/transcript/manifest.hpp

```diff
@@ -67,7 +67,6 @@ class Manifest {
         bool map_challenges;
     };
 
-    // TODO(luke): needed only in development; can be deleted when appropriate
```

Comment:
shouldn't this be here so we remember when to delete the manifest?

Reply:
Oh no this comment was literally just referring to the default constructor but its used in a bunch of places so this comment is just outdated.

Reply:
ah I see, my bad